server/plugin/webhook.go

```diff
@@ -893,6 +894,68 @@ func (p *Plugin) handleCommentAuthorNotification(event *github.IssueCommentEvent
 	p.sendRefreshEvent(authorUserID)
 }
 
+func (p *Plugin) handleCommentAssigneeNotification(event *github.IssueCommentEvent) {
+	author := event.GetIssue().GetUser().GetLogin()
+	assignees := event.GetIssue().Assignees
+
+	for _, assignee := range assignees {
+		userID := p.getGitHubToUserIDMapping(assignee.GetLogin())
+		if userID == "" {
+			continue
+		}
+
+		if author == assignee.GetLogin() {
+			return
+		}
+		if event.Sender.GetLogin() == assignee.GetLogin() {
+			return
+		}
+
+		authorUserID := p.getGitHubToUserIDMapping(author)
+		if authorUserID == "" {
+			return
+		}
+
+		assigneeID := p.getGitHubToUserIDMapping(assignee.GetLogin())
+		if assigneeID == "" {
```

Comment:
@hanzei  is this an acceptable way of testing for empty strings or do you have any suggestions ?

Reply:
Testing for `""` is the idiomatic way to test that in go.